homeassistant/components/blink/alarm_control_panel.py

```diff
@@ -57,8 +57,7 @@ def __init__(
     ) -> None:
         """Initialize the alarm control panel."""
         super().__init__(coordinator)
-        self.api: Blink = coordinator.api
-        self._coordinator = coordinator
+        self.api: Blink = self.coordinator.api
```

Comment:
When possible always prefer to access local variables over attributes. Local variables are cheaper to access than attributes.

We can use `coordinator` here instead of `self.coordinator`.

Reply:
Sorry - I knew that.

Reply:
https://github.com/home-assistant/core/pull/103430